zulipterminal/config/themes.py

```diff
@@ -52,149 +52,149 @@
 
 THEMES = {
     'default': [
-        (None,           'white',           ''),
-        ('selected',     'white',           'dark blue'),
-        ('msg_selected', 'white',           'dark blue'),
-        ('header',       'dark cyan',       'dark blue', 'bold'),
-        ('custom',       'white',           'dark blue', 'underline'),
-        ('content',      'white',           '',          'standout'),
-        ('name',         'yellow, bold',    ''),
-        ('unread',       'light blue',      ''),
-        ('active',       'light green',     ''),
-        ('idle',         'yellow',          ''),
-        ('offline',      'white',           ''),
-        ('inactive',     'white',           ''),
-        ('title',        'white, bold',     ''),
-        ('time',         'light blue',      ''),
-        ('bar',          'white',           'dark gray'),
-        ('help',         'white',           'dark gray'),
-        ('emoji',        'light magenta',   ''),
-        ('reaction',     'light magenta, bold', ''),
-        ('span',         'light red, bold', ''),
-        ('link',         'light blue',      ''),
-        ('blockquote',   'brown',           ''),
-        ('code',         'black',           'white'),
-        ('bold',         'white, bold',     ''),
-        ('footer',       'white',           'dark red',  'bold'),
-        ('starred',      'light red, bold', ''),
-        ('category',     'light blue, bold', ''),
-        ('unread_count', 'yellow',          ''),
+        (None,             'white',           ''),
+        ('selected',       'white',           'dark blue'),
+        ('msg_selected',   'white',           'dark blue'),
+        ('header',         'dark cyan',       'dark blue', 'bold'),
+        ('custom',         'white',           'dark blue', 'underline'),
```

Comment:
Could we use improvement for the `custom` style? Currently, it is a bit hard to tell what is 'custom' for.

Reply:
I think there are improvements still to be made, but some changes need more thought - they don't all have to be in one PR either - particularly with the change of spacing.